docs/remote_inference_blueprints/cohere_connector_chat_blueprint.md

```diff
@@ -126,7 +124,7 @@ Note: `conversation_id` here is the string value for "1", but you can use any st
 POST /_plugins/_ml/models/<MODEL_ID_HERE>/_predict
 {
   "parameters": {
-    "message": "What is the weather like in London?",
+    "message": "What is the weather like in London?"
```

Comment:
I tested this , the response is 
```
{
  "inference_results": [
    {
      "output": [
        {
          "name": "response",
          "dataAsMap": {
            "response_id": "b0f6eaf6-3f63-418b-9178-a5db8431e55e",
            "text": """Here is the current weather and forecast for London, UK:

**Current Weather:**

- Temperature: 26.0°C (79°F)
- Conditions: Cloudy with occasional sunny spells
- Humidity: 62%
- Wind: 3 mph (5 km/h) (NW)
- Visibility: 10 km (6 miles)

**Forecast for the next few days:**

The weather in London will continue to be moderately warm for the next few days with temperatures reaching 27°C (81°F) on Saturday. The coming days will be mostly cloudy with occasional sunny spells and chances of light rain throughout the weekend. 

Make sure to carry an umbrella if you are going outside, and consider using sunscreen as and when required. 

If you would like a more detailed or specific weather forecast for London, let me know, and I'll be happy to provide it!""",
            "generation_id": "89c59a14-f099-4100-95b0-7d0d68104f4b",
            "finish_reason": "COMPLETE",
            "token_count": {
              "prompt_tokens": 70,
              "response_tokens": 189,
              "total_tokens": 259,
              "billed_tokens": 248
            },
            "meta": {
              "api_version": {
                "version": "1"
              },
              "billed_units": {
                "input_tokens": 59,
                "output_tokens": 189
              }
            }
          }
        }
      ],
      "status_code": 200
    }
  ]
}
```

The result is different with the blueprint,  it doesn't have 
```
"documents": [
                ...
            ],
            "search_results": [
                ...
            ],
            "tool_inputs": null,
            "search_queries": [
                ...
            ]
```
Is that correct , @tianjing-li ?